### PR TITLE
feat(ui): show app version

### PR DESCRIPTION
**Pull request by rudolfs**
As specified in: https://www.figma.com/file/owmgsbs6lnUt8R1bixstCA/%F0%9F%8C%B1%F0%9F%86%99-Radicle-Upstream?node-id=4147%3A7518.

Screenshot 2020-09-14 at 18 08 34

### CODE REVIEW

File: native/ipc.js [hunk 1]
```diff
@@ -9,6 +10,8 @@ export const OPEN_PATH = "IPC_OPEN_PATH";
 //
 // The workaround is to use the electron native open dialog. As a bonus we
 // can configure it to allow users to create new directories.
+export const getVersion = () => window.electron.ipcRenderer.invoke(GET_VERSION);
```
Comment (xla) [inline native/ipc.js:13]: It looks like this function got in between the comment above that belongs to the function below.

**Review by xla — CHANGES_REQUESTED** [review]
Dope! Just one mix-up.

File: native/ipc.js (hunk 1 above)
Comment (rudolfs) [reply native/ipc.js]: Good catch!

Comment (rudolfs) [reply native/ipc.js]: https://github.com/radicle-dev/radicle-upstream/pull/901/commits/e85f82566712684a9e0f32e5e16c531a48c11192

**Review by xla — APPROVED** [review]
🍬 🔍 🛡 🌀

**Pull request merged**